core/modules/update/update.admin.inc

```diff
@@ -105,10 +105,10 @@ function update_settings_validate($form, &$form_state) {
  * Form submission handler for update_settings().
  *
  * Also invalidates the cache of available updates if the "Check for updates of
- * disabled modules, themes, and layouts" setting is being changed. The available updates
- * report needs to refetch available update data after this setting changes or
- * it would show misleading things (e.g., listing the disabled projects on the
- * site with the "No available releases found" warning).
+ * disabled and uninstalled modules, themes, and layouts." setting is being
+ * changed. The available updates report needs to refetch available update data
+ * after this setting changes or it would show misleading things (e.g., listing
+ * the disabled projects on the site with the "No available releases found" warning).
```

Comment:
...sorry to be a pain, but this line still exceeds the 80 chars limit. "warning)." needs to be on its own line.